src/SignalR/clients/csharp/Client/test/FunctionalTests/Hubs.cs

```diff
@@ -242,6 +248,14 @@ public static ChannelReader<int> StreamEchoInt(ChannelReader<int> source)
 
             return output.Reader;
         }
+
+        public static async IAsyncEnumerable<int> StreamIAsyncConsumer(IAsyncEnumerable<int> source)
+        {
+            await foreach(var item in source)
```

Comment:
Why the loop?

Reply:
You can't just return the IAsyncEnumerable here

Reply:
What am I missing?

Reply:
Nvm, I forgot to take out the async keyword when I tried this and the return errors that you need to use a yield return so that was a little misleading. Can definitely just return the source directly.